web/crux-ui/src/elements/dyo-input.tsx

```diff
@@ -46,6 +46,7 @@ export const DyoInput = forwardRef((props: DyoInputProps, ref: ForwardedRef<HTML
           name={name}
           ref={ref}
           disabled={disabled}
+          id={name}
```

Comment:
I've just noticed you are using this for tests.
That's something we're trying to avoid.
Please resolve those elements in the test some other way.